web-starter/index.js

```diff
@@ -1,55 +1,123 @@
 'use strict';
-var generators = require('yeoman-generator'), 
-  _ = require('lodash'),
-  Promise = require('bluebird'),
-  rp = require('request-promise'),
-  semver = require('semver'),
-  glob = Promise.promisify(require('glob')),
-  pkg = require('../package.json'),
-  GruntfileEditor = require('gruntfile-editor');
 
+var generators = require('yeoman-generator');
+var _ = require('lodash');
+var pkg = require('../package.json');
+var GruntfileEditor = require('gruntfile-editor');
+
+/**
+ * @type {Object<string, GruntFileEditor>}
```

Comment:
JSDoc notation for a generic type is `Object.<string, GruntFileEditor>` (mind the dot).